pepper-apis/src/main/java/org/broadinstitute/ddp/db/dto/ActivityInstanceDto.java

```diff
@@ -32,8 +33,8 @@ public ActivityInstanceDto(
             @ColumnName("participant_id") long participantId,
             @ColumnName("created_at") long createdAtMillis,
             @ColumnName("first_completed_at") Long firstCompletedAt,
-            @ColumnName("is_readonly") boolean isReadonly,
-            @ColumnName("is_hidden") boolean isHidden,
+            @ColumnName("is_readonly") Boolean isReadonly,
+            @ColumnName("is_hidden") Boolean isHidden,
```

Comment:
Can we keep this one as primitive `boolean` type?

Reply:
We need `Boolean` here because `ActivityInstanceDto.isReadonly` is used in the nullability check. However, if we have both `isReadonly()` and `getReadonly()` as discussed above, you can choose whatever you want. But the constructor parameter must be `Boolean` anyway.

Reply:
I understand the read-only, but why also change `isHidden`?

Reply:
Fixed.